tests/test_jams.py

```diff
@@ -519,6 +532,18 @@ def test_jobject_bad_field():
     jam.out_of_schema = None
 
 
+def test_jams_repr(input_jam):
```

Comment:
discussed offline -- since it seems to work offline, and the functionality isn't critical to anything downstream, we're okay with not being 100% thorough on validating the output of the repr/repr_html functions.